### DISCUSSION TITLE
add second hard disk in azuracast server for stations

### DISCUSSION THREAD

**Discussion by ebinvthomas** [AzuraCast/AzuraCast#6305] [Q&A]
I am currently facing an issue with setting up media storage on my server. The server consists of two drives, a system SSD (250GB) and an attached 2TB HDD. I have successfully installed and launched AzuraCast on the SSD, and everything is functioning properly. However, I am unable to locate the appropriate setting or configuration to direct the media storage to the 2TB HDD instead.

 It is likely that I am overlooking a crucial setting, and I kindly request your guidance on how to properly configure this setup.

Any assistance provided would be greatly appreciated.

**Comment by RadioFlamingo** [comment]
From your dashboard site, click administration, go to system settings, on the left click maintenance. from there you can see storage locations,

**Reply by ebinvthomas** [reply]
No !, It is not what we asking.

I said the server has two HDDs, and we installed OS on the main default HDD. 
We mounted it through SSH, and now on the panel, it's not showing. This means the second disk is not showing in the panel. 

That's why we asked how we can mount the second HDD to azuracast stations.

**Comment by Moonbase59** [comment]
@ebinvthomas Depends on how and where you mounted your drive. You also need the correct access permissions of course.

For instance, I have mounted my central music storage (NFS mount) under `/mnt/Musik` in the host, then set up 3 subfolders of it to be included into the station’s media folder `/var/azuracast/stations/testradio/media` within `/var/azuracast/docker-compose.override.yml` like so:

```yml
services:
  web:
    volumes:
      - /mnt/Musik/Tagged:/var/azuracast/stations/testradio/media/Tagged
      - /mnt/Musik/Schraeg:/var/azuracast/stations/testradio/media/Schraeg
      - /mnt/Musik/Other:/var/azuracast/stations/testradio/media/Other
```

The three folders then appear within AzuraCast’s default `media` folder as subfolders `Tagged`, `Schraeg` and `Other`, respectively.

You should be able to adapt this for mounting an entire drive. Under "Add Storage Location", just enter your mount path, i.e. `/mnt/Musik` and possibly a quota, and you should be set, assuming correct permissions. The main trick is that your docker container knows about the host mount. ("Local filesystem" means local to the _container_, _not_ local to the host.)

Additionally, you might have to set the station’s media storage location, under "Edit Profile → Administration".

HTH

**Reply by ebinvthomas** [reply]
We had already checked this possibility.  In the docker-compose.yml file, We could not locate the option "Add Storage Location".

**Reply by Moonbase59** [reply]
It’s under _Administration → System Maintenance → Storage Locations_, at least on my Rolling Release #[386578a](https://github.com/AzuraCast/AzuraCast/commit/386578ac12f29b118233e74239f46d1a40a025a0) (2023-06-02 13:07).

I’ve just rebuilt your situation, adding a spare drive as `/mnt/test` in my host, then setting

```yaml
services:
  web:
    volumes:
      - /mnt/test:/mnt/test
```
in `docker-compose.override.yml`, adding that (`/mnt/test`, the _right_ value in above file) as a new storage location, and setting a station to use this as its media folder (_Administration → Stations → (Station) Manage → Edit Profile → Administration → Media Storage Location_). and it works like a charm.

**Comment by cutratefuture** [comment]
this helped!
